### PR TITLE
Fix registration mail

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: app/core/views.py [hunk 1]
```diff
@@ -648,16 +649,16 @@ def send_mail(self, politician):
         )
         profile_url_absolute = self.request.build_absolute_uri(profile_url)
         send_mail(
-            _('Freedomvote account link'),
-            _("""
+            unicode(_('Freedomvote account link')),
+            dedent(unicode(_("""
             Hello,
 
             You receive the link for your profile on Freedomvote:
```
Comment (winged) [inline app/core/views.py]: Wording: Recipients may or may not know about Freedmovote when they receive this email, so I would add some short introduction here.

File: app/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -606,7 +607,7 @@ msgstr ""
 "            Hallo,\n"
 "\n"
 "            Sie erhalten den Link zu Ihrem Profil bei Freedomvote:\n"
```
Comment (winged) [inline app/locale/de/LC_MESSAGES/django.po]: Same as above - Wording

File: app/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -595,20 +596,20 @@ msgid ""
...
 "            The Freedomvote Team\n"
 "            "
 msgstr ""
 "\n"
-"             Bonjour,\n"
+"            Bonjour,\n"
 "\n"
-"             Vous recevez le lien pour votre profil sur Freedomvote:\n"
-"             {}/politician/{}/edit/profile/\n"
+"            Vous recevez le lien pour votre profil sur Freedomvote:\n"
```
Comment (winged) [inline app/locale/fr/LC_MESSAGES/django.po]: Same again, wording

File: app/core/views.py (hunk 1 above)
Comment (anehx) [reply app/core/views.py]: Yes they do. They just registrated themselves there...

Comment (winged) [reply app/core/views.py]: Okay, but still there should be a bit more explanation: "Keep this link and use it to login to your profile again" or similar

Comment (anehx) [reply app/core/views.py]: Seems fair.

**Pull request merged**